substrate/frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -108,6 +108,27 @@ impl From<FeasibilityError> for MinerError {
 	}
 }
 
+/// Reports the trimming result of mined solution
+#[derive(Debug, Clone)]
+pub struct TrimmingStatus {
+	weight: usize,
+	length: usize,
+}
+
+impl TrimmingStatus {
+	pub fn is_trimmed(&self) -> bool {
+		self.weight + self.length > 0
```

Comment:
nit: small opt to not rely on the addition and avoid potential edge-cases with overflow 
```suggestion
		self.weight > 0 ||  self.length > 0
```